### PR TITLE
Created user entity

### LINKED ISSUE #1: Create Entity User
-> `\src\Domain\Entity\User.php`

- id (uuid)
- name (string 100)
- email (string 100, unique)
- password (string 255)
- status (boolean)
- photo (string 255)
- roles (json/array)
- createdAt (datetime)
- updatedAt (datetime)
- deletedAt (datetime, null)

### PR DESCRIPTION
**Pull request by mariaclaraabreu**
Resolve #1

### CODE REVIEW

File: src/Entity/User.php [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $password;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private boolean $status;
```
Comment (alessandrofeitoza) [inline src/Entity/User.php]: change to `private bool $status`

File: src/Entity/User.php [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $photo;
+
+    /**
+     * @ORM\Column(type="json")
+     */
+    private $roles = [];
```
Comment (alessandrofeitoza) [inline src/Entity/User.php]: change to `private array $roles = []`, because array is a type for $roles

File: src/Entity/User.php [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+     * @ORM\Column(type="datetime")
+     */
+    private DateTime $updatedAt;
+    
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private DateTime $deletedAt;
+
+    public function getId(): string
```
Comment (alessandrofeitoza) [inline src/Entity/User.php]: the type return correct is `Uuid`

File: src/Entity/User.php [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+    {
+        return $this->id;
+    }
+
+    public function setId($id): void
```
Comment (alessandrofeitoza) [inline src/Entity/User.php]: missing parameter type, for example:
`public function setId(Uuid $id): void`

**Review by alessandrofeitoza — APPROVED** [review]

**Review by pamplona007 — APPROVED** [review]

**Pull request merged**